ncm-ssh/src/main/perl/ssh.pm

```diff
@@ -110,7 +129,9 @@ sub Configure
     my $ok         = 1;
 
     if ($ssh_config->{daemon}) {
-        if ($self->handle_config_file(SSHD_CONFIG, 0600, $ssh_config->{daemon}, \&valid_sshd_file))
+        if ($self->handle_config_file($ssh_config->{daemon}->{config_path}
+                                          || DEFAULT_SSHD_CONFIG,
+                                      0600, $ssh_config->{daemon}, \&valid_sshd_file))
         {
             CAF::Process->new([qw(/sbin/service sshd condrestart)], log => $self)->run();
```

Comment:
this is not EL7 safe, systemd only has `try-restart`, better to switch to `CAF::Service` here (does this even work on solaris?)